Add docu for undocumented functions

deining opened the pull request:
This PR adds doucmentation for string manipulation functions, duration functions and the `crypto.FNV32a` function.
This PR closes #1898.
This PR closes #181.
This PR closes #739.

jmooring commented on content/en/functions/strings.ToLower.md:
```diff
@@ -0,0 +1,14 @@
+---
```
The already documented "lower" function is an alias for strings.ToLower. You can remove this page.

You are welcome to add a second signature to the existing function. Here's an example:
https://gohugo.io/functions/replacere

I keep this link handy to find aliases: https://raw.githubusercontent.com/gohugoio/hugo/master/docs/data/docs.json

deining replied:
Page removed and second signature added to the existing function.

jmooring commented on content/en/functions/strings.ToUpper.md:
```diff
@@ -0,0 +1,14 @@
+---
```
The already documented "upper" function is an alias for strings.ToUpper. You can remove this page.

You are welcome to add a second signature to the existing function.

deining replied:
Page removed and second signature added to the existing function.

jmooring reviewed (changes requested):
Thank you for this!

jmooring commented:
One more thing... your commit is not addressing issues, it is closing them!  

Closes #181
Closes #739
Closes #1898

deining commented on content/en/functions/crypto.FNV32a.md:
```diff
@@ -0,0 +1,16 @@
+---
+title: crypto.FNV32a
+description: Calculate the FNV (Fowler–Noll–Vo) 32 bit hash of a given string.
```
Changed as requested.

deining commented on content/en/functions/duration.md:
```diff
@@ -0,0 +1,56 @@
+    {{ printf "There are %.0f seconds in one day." (duration "hour" 24).Seconds }}
+    <!-- Output: There are 86400 seconds in one day. -->
+
+Also, you can use the `duration` function with [Hugo pipes](https://gohugo.io/hugo-pipes/introduction/):
```
Replaced the sentence as requested.

deining commented on content/en/functions/crypto.FNV32a.md:
```diff
@@ -0,0 +1,16 @@
+
+This function calculates the 32 bit [FNV1a hash](https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function#FNV-1a_hash) of a given string according to the [specification](https://datatracker.ietf.org/doc/html/draft-eastlake-fnv-12):
+
+    {{ crypto.FNV32a "Hello world" }} → "1498229191"
```
This is corrected now.

deining commented on content/en/functions/duration.md:
```diff
@@ -0,0 +1,56 @@
+    {{  mul 120 60 | mul 1000 | duration "millisecond"}} → 2h0m0s
+
+You have to specify a time unit for the number given to the function. Valid time units are:
+* nanoseconds,
```
Replaced the example with a table, as requested.

deining commented on content/en/functions/strings.Contains.md:
```diff
@@ -0,0 +1,24 @@
+---
+title: strings.Contains
+description: Check if a string contains a substring.
```
Changed as requested.

deining commented on content/en/functions/duration.md:
```diff
@@ -0,0 +1,56 @@
+    parent: "functions"
+keywords: [time duration]
+signature: ["duration TIME_UNIT"]
+hugoversion:
```
Line removed as requested.

deining commented on content/en/functions/strings.Contains.md:
```diff
@@ -0,0 +1,24 @@
+    parent: "functions"
+keywords: [string strings substring contains]
+signature: ["strings.Contains STRING SUBSTRING"]
+hugoversion: ""
```
Line removed as requested.

deining commented on content/en/functions/duration.md:
```diff
@@ -0,0 +1,56 @@
+Also, you can use the `duration` function with [Hugo pipes](https://gohugo.io/hugo-pipes/introduction/):
+
+    {{  mul 7.75 60 | duration "minute"}} → 7h45m0s
+    {{  mul 120 60 | mul 1000 | duration "millisecond"}} → 2h0m0s
```
Spacing corrcted as requested.

deining commented on content/en/functions/strings.ContainsAny.md:
```diff
@@ -0,0 +1,24 @@
+---
+title: strings.ContainsAny
+description: Check if a string contains any character from a given string.
```
Changed as requested.

deining commented on content/en/functions/strings.ContainsAny.md:
```diff
@@ -0,0 +1,24 @@
+    parent: "functions"
+keywords: [string strings substring contains any]
+signature: ["strings.ContainsAny STRING CHARACTERS"]
+hugoversion: ""
```
Removed line as requested.

deining commented on content/en/functions/strings.Contains.md:
```diff
@@ -0,0 +1,24 @@
+
+    {{ strings.Contains "Hugo" "Go" }} → false
+
+Numbers can be handled as well:
```
Section removed as requested.

deining commented on content/en/functions/strings.ContainsAny.md:
```diff
@@ -0,0 +1,24 @@
+
+    {{ strings.ContainsAny "Hugo" "Gm" }} → false
+
+Numbers can be handled as well:
```
Section removed as requested.

deining commented:
> One more thing... your commit is not addressing issues, it is closing them!
> 
> Closes #181 Closes #739 Closes #1898

I edited the description of the PR, these three issues are now linked with this PR and will be closed once this PR was merged.

deining commented on content/en/functions/crypto.FNV32a.md:
```diff
@@ -0,0 +1,16 @@
+    parent: "functions"
+keywords: [crypto hash FNV32 Fowler-Noll-Vo]
+signature: ["crypto.FNV32a STRING"]
+hugoversion:
```
Line was remooved as requested.
